app/models/Event.js

```diff
@@ -47,27 +47,22 @@ const eventSchema = new Schema({
       }
     }
   },
-  rsvp: {
-    yes: {
-      type: Schema.Types.ObjectId,
-      ref: 'User'
-    },
-    no: {
-      type: Schema.Types.ObjectId,
-      ref: 'User'
-    },
-    mayBe: {
-      type: Schema.Types.ObjectId,
-      ref: 'User'
-    }
-  },
+
+  rsvp_yes: [{
+    type: mongoose.Types.ObjectId
```

Comment:
This should be referenced to User schema show that instead of searching by those ids, we can get the data by simply deep populations